Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -209,7 +204,7 @@ PixelShaderUid GetPixelShaderUid()
     warn_once = false;
   }
 
-  if (uid_data->per_pixel_lighting)
+  if (g_ActiveConfig.bEnablePixelLighting)
```

Comment:
It is expected that the UID itself depends on the host config? I guess the expected behavior is that the shader depends on the UID and on the host state, but the UID shall not depend on the host state. Bad luck, per pixel lighting goes in another way...